setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import setup
+
+setup(
+    name='wikiextractor',
+
+    description='A script that extracts and cleans text from a Wikipedia'
+                'database dump',
+    author='Giuseppe Attardi',
```

Comment:
@attardi I am not sure if you are the author but if you are not you might change this keyword and you might move yourself to `maintainer` and `maintainer_email`.